src/parser/mod.rs

```diff
@@ -10816,7 +10836,7 @@ impl<'a> Parser<'a> {
         self.expect_token(&Token::LParen)?;
         let aggregate_functions = self.parse_comma_separated(Self::parse_aliased_function_call)?;
         self.expect_keyword(Keyword::FOR)?;
-        let value_column = self.parse_object_name(false)?.0;
+        let value_column = self.parse_period_separated_identifiers()?;
```

Comment:
```suggestion
        let value_column = `self.parse_period_separated(|p| p.parse_identifier(false))?;
```
Can we inline the code? thinking since the current method is only a wrapper around it